src/index.js

```diff
@@ -383,8 +378,8 @@ class MiniCssExtractPlugin {
                       ])
                     : '',
                   `var insert = ${insert};`,
-                  `if (typeof insert === 'function') { insert(linkTag); }`,
-                  `else { var target = document.querySelector(${insert}); target && insert === 'body' ? target && target.insertBefore(linkTag,target.firstChild) : target.appendChild(linkTag); } `,
+                  `if (insert) { insert(linkTag); }`,
+                  `else {var head = document.getElementsByTagName("head")[0]; head.appendChild(linkTag);} `,
```

Comment:
Can we simplify that, we can generate runtime based on type of option and reduce size of runtime code

Reply:
So pretty much append the correct string of code based on the option a compile time. Rather than logical operators at runtime

Reply:
Done